n3fit/runcards/examples/nnpdf40-like.yml

```
@@ -131,15 +129,16 @@ parameters: # This defines the parameter dictionary that is passed to the Model
 
 fitting:
   fitbasis: EVOL
+  savepseudodata: False
```

Why did you add this?

---

Usually we don't need it... but yes maybe better to have it true by default...
```suggestion
  savepseudodata: True
```

---

The default is true if it's not set explicitly in the runcard, at some point it was changed to be stored by default because it's not that costly and reproducing it isn't stable at all. And I believe we had some problem because we didn't have this information for some fits

---

I rather have the flag in the example even if set to the default value so that I can set to false without writing `savepsudodata` or something like that ^^U